pinc/ProjectState.inc

```diff
@@ -10,12 +10,12 @@ class ProjectState
     public string $star_metal;
 
     public function __construct(
-        $name,
-        $medium_label,
-        $label,
-        $forum,
-        $phase,
-        $star_metal
+        string $name,
+        string $medium_label,
+        string $label,
+        string $forum,
```

Comment:
`$forum` is interesting because it really should be an integer. For some reason we're putting them in `''`s in `site_vars.php.template` (see [line 56](https://github.com/DistributedProofreaders/dproofreaders/blob/master/pinc/site_vars.php.template#L56)) but they're 100% integers because of phpBB.

I think the Right Thing To Do is to remove the `''`s from the `site_vars.php.template` file and add a comment in `SETUP/configuration.sh` before those about them needing to be integers, then having `get_forum()`'s return type be an int. I'm also amenable to that being a fast-follow PR but I feel like in this case we're passing up an opportunity to fix this -- and it seems pretty surgical?